recipe-server/normandy/recipes/management/commands/update_actions.py

```diff
@@ -133,6 +133,8 @@ def get_arguments_schema_by_implementation(action_name, implementation_directory
 
 
 def get_arguments_schema_by_schemas(action_name, schemas, aliases):
+    if action_name in schemas:
+        return schemas[action_name]
```

Comment:
I get what the code does and how it works but a couple of months from now someone who sees this code is going to have to stop and wonder and ponder "Why this? Why not allow the alias lookup? Why was this done? hmm"

Reply:
I think that this makes sense when taken as a whole. It does the direct thing of looking up the key in the schemas, and then falls back to the aliases "in the case it's called something else in the schemas.json file".

The reason for this change is because I renamed `consoleLog` to `console-log` in the schema json. That caused this function to fail, since it would always try and use the alias.

Reply:
Ah, I see. 

Can we edit the `settings.ACTION_ALIASES` thing to be an empty dict now?